woocommerce/woocommerce
# CES modal modifications

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR introduces several modifications to the CES modal.

![Screenshot 2023-06-09 at 16 51 02](https://github.com/woocommerce/woocommerce/assets/1314156/0d0080de-326e-4faa-b544-5072a49350fa)

**Acceptance criteria**
- [ ] Remove the modal description.
- [ ] Add the fields `Additional thoughts` and `Your email address`. Both are optional.
  - [ ] Add an email validation for the second field.
  - [ ]Validate the `Additional thoughts` field to ensure it is smaller than 500 characters. This wasn't in the original requirements, but I thought it would be good to add it.
- [ ] Modify the first question from `The product editing screen is easy to use` to `PRODUCT FORM IS EASY TO USE`.
- [ ] Modify the second question from `The product editing screen's functionality meets my needs` to `PRODUCT FORM’S FUNCTIONALITY MEETS MY NEEDS`.
- [ ] The border and background when havering an icon now look different.
- [ ] The icon's height is now smaller.
- [ ] The distance between each icon now is smaller too.

Closes #38502.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch, and enable the blocks product management experience
2. Go to WooCommerce > Add New and press `Share feedback` in the bottom bar.

![Screenshot 2023-06-11 at 22 09 56](https://github.com/woocommerce/woocommerce/assets/1314156/9b9a3d98-df57-4f98-b30a-837452d12664)

3. The CES modal should be displayed.
4. Verify that the validation is working correctly for the fields.
5. Confirm that the score is being recorded correctly.
6. Also click the three dots next to the `Add` button in the header and select `Share feedback`. The same form should be shown.

![Screenshot 2023-06-11 at 22 15 36](https://github.com/woocommerce/woocommerce/assets/1314156/1c8043c3-a1c9-4153-b6ac-9d0f95af56e0)

// cc @jarekmorawski
